PSI_Project/Models/Comment.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace PSI_Project.Models;
+
+public class Comment : BaseEntity //name is not needed for Comment. Is ID needed? (inherited BaseEntity because it is required so CommentRepository could inherit BaseRepository)
```

Comment:
I think it is not required but I would leave it in case we will later would want to add some functionality like "reply" or smth like this

Reply:
agreed